doc/faq.rst

```diff
@@ -181,86 +181,60 @@ Using Spyder
 .. dropdown:: Q: How do I get Spyder to work with my existing Python packages/environment?
    :container: + dropdown-id-using-existing-environment
 
-   To work with an existing environment in Spyder, you need to change Spyder’s default Python interpreter.
+   To work with an existing environment in Spyder, change the default Python interpreter for new :doc:`/panes/ipythonconsole`\s to point to this environment.
    To do so, click the name of the current environment in the status bar, and then click :guilabel:`Change default environment in Preferences`.
 
    .. image:: /images/faq/faq-change-environment.png
       :alt: Change default environment in Preferences option in status bar
 
-   This will open the :guilabel:`Preferences` dialog in the :guilabel:`Python interpreter` section.
-   Here, select the option :guilabel:`Use the following Python interpreter`, and use the dropdown below to select your preferred environment.
-   If its not listed, use the text box or the :guilabel:`Select file` button to enter the path to the Python interepreter you want to use.
-   See the :doc:`/panes/ipythonconsole` for more information.
+   This will open the :guilabel:`Python interpreter` pane of Spyder's preferences.
+   Here, select the option :guilabel:`Use the following Python interpreter`, and use the dropdown below to select your preferred environment, or paste/browse for the path to your desired Python interpreter.
```

Comment:
I don't like the "paste/browse for" construction, but I am not sure I know something better. This is the best I could come up with:

```suggestion
   Here, select the option :guilabel:`Use the following Python interpreter`, and use the dropdown below to select your preferred environment.
   If your environment is not listed, you can also enter the path to your desired Python interpreter directly as explained in the note below.
```
The last six words may be superfluous. I added them because readers may miss the note because they don't look under the picture.

Reply:
Yeah, I didn't like it either and previously had something similar; I just also didn't t want to duplicate what was stated again immediately below, especially in the context of a relatively concise FAQ answer.

What about something a little more succinct which links the note? 

```suggestion
   Here, select the option :guilabel:`Use the following Python interpreter`, and use the dropdown below to select your preferred environment.
   If it's not listed, see :ref:`the note below <faq-env-not-listed>`.
```